osu.Game.Rulesets.Mania/UI/Components/SpecialColumnType.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Mania.UI.Components
+{
+    public class SpecialColumnType : ColumnType
+    {
+        public override Color4 ColumnColour => new Color4(0, 48, 63, 255);
+        public override float DefaultColumnWidth => 70;
```

Comment:
I'd expose and reuse
https://github.com/ppy/osu/blob/81d2487f65f827a35b124719507d641e1da07285/osu.Game.Rulesets.Mania/UI/Column.cs#L25

Reply:
Is there any reason to still put it in `Column`?

Reply:
IMO the two constants should be side by side for easy searching and access.